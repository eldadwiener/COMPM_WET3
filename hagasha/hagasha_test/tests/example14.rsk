<header>
<unimplemented>
<implemented> main,1
</header>
PRNTC 10
PRNTC 84
PRNTC 101
PRNTC 115
PRNTC 116
PRNTC 58
PRNTC 32
PRNTC 99
PRNTC 104
PRNTC 101
PRNTC 99
PRNTC 107
PRNTC 32
PRNTC 34
PRNTC 40
PRNTC 66
PRNTC 69
PRNTC 88
PRNTC 80
PRNTC 41
PRNTC 34
PRNTC 32
PRNTC 10
COPYI I3 2
COPYI I4 3
SLETI I5 I3 I4
BNEQZ I5 29
UJUMP 43
PRNTC 50
PRNTC 32
PRNTC 60
PRNTC 32
PRNTC 51
PRNTC 32
PRNTC 103
PRNTC 111
PRNTC 111
PRNTC 100
PRNTC 33
PRNTC 33
PRNTC 10
UJUMP 55
PRNTC 50
PRNTC 32
PRNTC 62
PRNTC 32
PRNTC 51
PRNTC 32
PRNTC 98
PRNTC 97
PRNTC 100
PRNTC 33
PRNTC 33
PRNTC 10
COPYI I5 2
COPYI I6 3
SGRTI I7 I5 I6
BNEQZ I7 60
UJUMP 73
PRNTC 50
PRNTC 32
PRNTC 62
PRNTC 32
PRNTC 51
PRNTC 32
PRNTC 98
PRNTC 97
PRNTC 100
PRNTC 33
PRNTC 33
PRNTC 10
UJUMP 86
PRNTC 50
PRNTC 32
PRNTC 60
PRNTC 32
PRNTC 51
PRNTC 32
PRNTC 103
PRNTC 111
PRNTC 111
PRNTC 100
PRNTC 33
PRNTC 33
PRNTC 10
COPYI I2 I1
RETRN
HALT
