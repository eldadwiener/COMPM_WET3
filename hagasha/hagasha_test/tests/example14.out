
Test: check "(BEXP)" 
2 < 3 good!!
2 < 3 good!!
Reached Halt.
